Files/Views/MainPage.xaml.cs

```diff
@@ -28,6 +28,8 @@ namespace Files.Views
     /// </summary>
     public sealed partial class MainPage : Page, INotifyPropertyChanged
     {
+        public static MainPage Instance => (MainPage)((Frame)Window.Current.Content).Content;
```

Comment:
Is this change related to the auto preview of readme files?

Reply:
Can be removed